packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -92,9 +92,9 @@ export const Timeline: React.FC<TimelineProps> = ({
   hideToolbar = false,
   isFullWidth = false,
   startDate,
-  endDate,
-  today,
-  range,
+  endDate = null,
+  today = startOfToday(),
```

Comment:
Waiting for confirmation from @k9dh3zij if this should be the start of the day or midday (affects where the marker appears).

Reply:
Let's go for midday for now, please. I've raised #3095 to look into this further, as I think there are better ways of indicating the current day